processor.py

```diff
@@ -94,11 +94,11 @@ def det_recur(self, A):
             v = A[0][0] * A[1][1] - A[1][0] * A[0][1]
             return v
         for c in range(len(A)):
-            d += (-1 if (c%2) else 1) * A[0][c] * self.det_recur(self.get_minor(A, 0, c))
+            d += (-1 if c % 2 else 1) * A[0][c] * self.det_recur(self.get_minor(A, 0, c))
         return d
 
     def determinant(self):
-        if (self.R == self.C):
+        if self.R == self.C:
```

Comment:
This change is fine, since it looks like the rest of the script doesn't use brackets.